Jenkinsfile

```diff
@@ -42,9 +48,12 @@ pipeline {
         }
       }
       steps {
-        sshagent (credentials: ['4ff09dce-407b-41d3-847a-9e6609dd91b8']) {
-          sh "npm run release"
-          sh "git push --follow-tags origin ${BRANCH_NAME}"
+        withCredentials([
+            usernamePassword(credentialsId: 'a7ac7f84-64ea-4483-8e66-bb204484e58f', passwordVariable: 'GIT_PASSWORD', usernameVariable: 'GIT_USER')
+        ]) {
+          sh "git fetch"
```

Comment:
I think this should not work, Jenkins has no `.git/` folder

Reply:
okay, if you say so I trust you

Reply:
No need to trust :D this run was successful: https://jenkins.mesosphere.com/service/jenkins/job/Frontend/job/mesosphere-mockserver-pipeline/job/jgieseke%252Fdocker-ci/12/console (I deleted the tag from github already to prevent chaos)